tests/_source/index.tsx

```diff
@@ -159,16 +159,17 @@ test.serial('render svg', t => {
 	);
 
 	t.truthy(element);
-	t.true(document.createElementNS.calledTwice);
+	t.true(createElementNSSpy.calledTwice);
 
 	const xmlns = 'http://www.w3.org/2000/svg';
-	t.deepEqual(document.createElementNS.firstCall.args, [xmlns, 'text']);
-	t.deepEqual(document.createElementNS.secondCall.args, [xmlns, 'svg']);
+	t.deepEqual(createElementNSSpy.firstCall.args, [xmlns, 'text']);
+	t.deepEqual(createElementNSSpy.secondCall.args, [xmlns, 'svg']);
+	createElementNSSpy.restore();
```

Comment:
Suggestions wanted to avoid having to manually create and restore each spy explicitly.